components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue

```diff
@@ -19,7 +19,7 @@
         v-slot="props"
         width="20%"
         field="meta"
-        :label="$t('tabs.tabActivity.price')">
+        :label="`${$t(`tabs.tabActivity.price`)} (KSM)`">
```

Comment:
not sure is it need to hardcode `KSM` here?

Reply:
isnt price always in ksm here?
can anyone confirm? @roiLeo @exezbcz

Reply:
it would be better to not hardcode currency (futur MOON integration)

Reply:
Yes, it is currently the only currency, but it would be better if it were more adaptable for future use.

Reply:
okay replaced with variable